scripts/release_scripts/update_configs.py

```diff
@@ -144,22 +144,29 @@ def check_updates_to_terms_of_service(
                 f.write(line)
 
 
-def verify_feconf(release_feconf_path, verify_mailgun_api):
+def verify_feconf(release_feconf_path, verify_email_api_keys):
```

Comment:
Why does the additional arg need to be a flag? Where is the flag used?

Reply:
Ah wait, scratch that -- I see, it used to exist already and you just changed the name. Please ignore this thread then.